init-tools.cmd

```diff
@@ -26,8 +26,6 @@ if exist "%BUILD_TOOLS_SEMAPHORE%" (
   goto :EOF
 )
 
-if exist "%TOOLRUNTIME_DIR%" rmdir /S /Q "%TOOLRUNTIME_DIR%"
```

Comment:
Also, might be nice while you're in here to change:

    if [%1]==[force] ( 

to 

    if /I [%1]==[force] (